wikiclass/feature_lists/wikidatawiki.py

```diff
@@ -21,6 +32,164 @@ class items:
     """
     HUMAN = 'Q5'
 
+def _process_item_doc(text):
+    if text is not None:
+        return json.loads(text)
+    else:
+        return None
+
+def _process_item(item_doc):
+    item = pywikibase.ItemPage()
+    item.get(content=item_doc or {'aliases': {}})
+    return item
```

Comment:
Maybe what you meant is revscoring.features.wikibase.datasources.revision_oriented